iam/api-client/quickstartv2.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+#
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START iam_quickstartv2]
+import os
+
+from google.oauth2 import service_account
+import googleapiclient.discovery
+
+
+def quickstart():
```

Comment:
I think you should add argument for better testability:

```python
def quickstart(project_id, member):
```

Then you can add TODOs in `if __name__ == '__main__':` clause below.

Reply:
Done